iamspanner/server.go

```diff
@@ -28,7 +29,11 @@ type IAMServer struct {
 
 // ServerConfig configures a Spanner IAM policy server.
 type ServerConfig struct {
+	// ErrorHook is called when errors occur in the IAMServer.
 	ErrorHook func(context.Context, error)
+	// ValidateMember is a custom IAM member validator.
+	// When not provided, iammember.Validate will be used.
+	ValidateMember func(string) error
```

Comment:
Should we consider passing `context.Context` here as well, in order to accommodate checking that the member exists via a network call?

Reply:
I changed my mind! Let's assume that we still only validate the format of the member - otherwise with upstream network calls, we would need to provide some type of batch API. We could still add that separately!